core/src/com/unciv/ui/worldscreen/mainmenu/OptionsPopup.kt

```diff
@@ -144,6 +148,28 @@ class OptionsPopup(val previousScreen:CameraStageBaseScreen) : Popup(previousScr
         innerTable2.add(previousScreen.game.version.toLabel()).pad(10f).row()
     }
 
+    private fun addMinimapSizeSlider() {
+        innerTable2.add("Show minimap".tr())
+
+        val minimapSliderLimit = resolutionArray.indexOf(settings.resolution) + 1
+        val minimapSlider = Slider(0f, minimapSliderLimit.toFloat(), 1f, false, skin)
+        minimapSlider.value = if(settings.showMinimap) min(settings.minimapSize, minimapSliderLimit).toFloat() else 0f
+        minimapSlider.onChange {
+            val size = minimapSlider.value.toInt()
+            if (size == 0) {
+                settings.showMinimap = false
+            } else {
+                settings.showMinimap = true
+                settings.minimapSize = size
+            }
+            settings.save()
+            Sounds.play(UncivSound.Click)
```

Comment:
This should be automatically done due to the .onChange{}, no?

Reply:
Huh? I often get needlessly confused, but in this case - _what_ gets done automatically? Those lines are within the .onChange lambda, onChange is our helper doing nothing but wrapping addListener, and upstream from there it's all libGdx? I admit I simply copied the other sliders...

Reply:
The 'click' sound... My mistake, it's only for onClick not onChange